Translations/WinMerge/French.po

```diff
@@ -841,7 +841,7 @@ msgid "&View Whitespace"
 msgstr "Afficher les &espaces"
 
 msgid "View E&OL"
-msgstr "Afficher fin de ligne (E&OL)"
+msgstr "Afficher les carractères de &fin de ligne (EOL)"
```

Comment:
Petite faute de frappe : carractères au lieu de caractères (avec un seul r)

Reply:
Merci pour la correction, et merci pour les mercis

Thanks for the correction, and thanks for the thanks